client/post-editor/editor-post-formats/accordion.jsx

```diff
@@ -1,68 +1,64 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	pick = require( 'lodash/pick' ),
-	find = require( 'lodash/find' ),
-	classNames = require( 'classnames' ),
-	isEmpty = require( 'lodash/isEmpty' );
+import React, { PropTypes } from 'react';
+import { has, isEmpty } from 'lodash';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
-var Accordion = require( 'components/accordion' ),
-	Gridicon = require( 'components/gridicon' ),
-	siteUtils = require( 'lib/site/utils' ),
-	PostFormats = require( './' );
+import Accordion from 'components/accordion';
+import Gridicon from 'components/gridicon';
+import siteUtils from 'lib/site/utils';
+import PostFormats from './';
 
-module.exports = React.createClass( {
+export default React.createClass( {
 	displayName: 'EditorPostFormatsAccordion',
 
 	propTypes: {
-		site: React.PropTypes.object,
-		post: React.PropTypes.object,
-		postFormats: React.PropTypes.arrayOf( React.PropTypes.shape( {
-			slug: React.PropTypes.string,
-			label: React.PropTypes.string
-		} ) )
```

Comment:
I realized pretty late in reviewing this that the object structure of the reducer is slug to label string, not slug to an object of properties, and while that makes it a little less obvious, I'd still stand by my previous remarks. Interesting discussion in any case 😄